WordPress/Classes/ViewRelated/Comments/CommentViewController.m

```diff
@@ -660,6 +659,30 @@ - (void)textView:(UITextView *)textView didTypeWord:(NSString *)word
     [self.suggestionsTableView showSuggestionsForWord:word];
 }
 
+- (void)updateUIForExpandedReply
+{
+    [[self view] bringSubviewToFront:[self replyTextView]];
+    [UIView animateWithDuration:self.replyTextView.animationDuration animations:^{
+        [[self navigationController] setNavigationBarHidden:YES];
```

Comment:
You switched some other cases to use `setNavigationBarHidden:animated:` but not here (or in the following method) – not sure if you missed these or not?

Reply:
I did miss those, that display had the fewest issues so it didn't get as much attention.  This will be corrected in the next push.